src/components/PagePaginationLink/PagePaginationLink.module.css

```diff
@@ -0,0 +1,43 @@
+.pagePagination {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: flex-end;
+  background-color: var(--light-04);
+  width: 100%;
+  border: 1px solid var(--primary-700);
+  border-radius: 16px;
+  padding: 1rem 1.5rem 1.5rem;
+  gap: 1rem;
+}
+
```

Comment:
There is meant to be an hover effect but can't find any. I shared a figma design to you showing different state for this.

here is the link again

https://www.figma.com/file/9eqpfWUrL3gAQ6Royvl75u/Blockchain-Education?node-id=10267%3A64557#236668373